domain/schema_validator.go

```diff
@@ -141,6 +133,7 @@ func hasRelatedTableID(relatedTableIDs, updateTableIDs []int64) bool {
 
 func (s *schemaValidator) isAllExpired(txnTS uint64) bool {
 	if s.latestSchemaInfo == nil {
+		log.Infof("the schema validator stopped before judging")
```

Comment:
We will get two log info if lastestSchemaInfo == nil, one says "validator stopped before judging", then the other says "validator's latest schema is expired". It may confuse the user.